src/modules/NtpClient.rb

```diff
@@ -1090,7 +1090,7 @@ def new_write_progress
 
     def update_cfa_record(record)
       cfa_record = record["cfa_record"]
-      cfa_record.value = record["address"]
+      cfa_record.value = record["address"].strip
```

Comment:
I think it should be handled even more upper level, record should not contain newline. so probably somewhere near autoyast import.